_use_cases/encryption-at-rest.markdown

```diff
@@ -0,0 +1,105 @@
+---
+name: Encryption At Rest
+---
+
+## Why
+
+Apache Kafka&#174; does not directly support any form of encryption for data stored within a broker. This means that the contents
+of records sent to Apache Kafka are stored in the clear. Anyone with sufficient privileges, such as a Kafka Administrator
+with file system access, is able to read the contents of the records.
+
+This present a challenge for an enterprise using Kafka to distribute confidential data or PII (Personally Identifiable
+Information).  Enterprises are often subject to confidential requirements coming from governmental bodies such as
+GDPR in the EU or LGPD in Brazil, industry standards such as HIPAA in the health domain or PCI/DSS in the payments domain,
+in addition to any in-house security and compliance requirements.
+
+The problem is made more complex if the enterprise has opted to utilise a Cloud Kafka Service as the confidential
+data is now residing in the clear on the file systems of the service provider.
```

Comment:
This is a specific example of a problem that can exist even without cloud in the picture. It's simply about the trust boundaries within an organisation. If compliance requirements dictate that the people (internal or external) who run the Kafka cluster (and therefore have broker access) should not be trusted to see the business data being stored there (e.g. by some other business function in the org) then essentially the same problem exists. 

If we're aiming this at decision makers then let's not give them the excuse of misunderstanding and thinking "we run Kafka on-prem, so this doesn't apply to us". I.e. describe the general problem and use cloud only as ac concrete example of it.

Reply:
I've run out of steam on this one today.  I can quite find a formulation of words that I'm happy with.  I'll take a look tomorrow.

Reply:
Raised as #32